internal/chart/cli/cli.go

```diff
@@ -0,0 +1,23 @@
+package cli
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/weka/gohomecli/internal/cli/app"
+)
+
+func init() {
+	app.AppCmd.AddGroup(&cobra.Group{
```

Comment:
one fix I'd added in my PR: child has zero knowledge about parent due to cyclic dependencies. So I decided to create `lazy init` https://github.com/weka/gohomecli/pull/9/files#diff-8a64b5d011cf2f10abe303a474991c89bd1427abb4a1a0bbe1d76de8b4be9e4cR5